packages/frontend/src/components/VisualizationConfigs/TableConfigPanel/GeneralSettings.tsx

```diff
@@ -1,5 +1,5 @@
-import { Checkbox, FormGroup } from '@blueprintjs/core';
-import { Box, Title, Tooltip } from '@mantine/core';
+import { FormGroup } from '@blueprintjs/core';
```

Comment:
do we use FormGroup anywhere?

Reply:
The drag and drop parts are still blueprint. Now that you did the z-index thing I think I can convert them too.

Reply:
I looked a little more closely. It's still wrapping the column config thing that is in Blueprint. I'm planning to port that in the next patch.

Reply:
perfect